src/classes/TDTM_TriggerHandler.cls

```diff
@@ -36,7 +36,7 @@
 * @description Class called from each of the triggers (only one per object) that follow the TDTM design. It's in 
 * charge of figuring out which of the classes that do the actual work need to be called, and calling them.
 */
-public with sharing class TDTM_TriggerHandler {
```

Comment:
Classes and triggers should use the `UpperCamelCase` naming convention. I suggest to rename accordingly.